packs/familiar-abilities/absorb-familiar.json

```
@@ -0,0 +1,33 @@
+{
+    "_id": "iBZzOaRNodGtfMVJ",
+    "img": "systems/pf2e/icons/actions/Passive.webp",
+    "name": "Absorb Familiar",
+    "system": {
+        "actionType": {
+            "value": "passive"
+        },
+        "actions": {
+            "value": null
+        },
+        "category": "familiar",
+        "description": {
+            "value": "<p>Your familiar can transform into a mark you carry on your flesh, typically seeming like a birthmark, tattoo, or gem that vaguely resembles its normal form. When transformed, the familiar can’t act except to turn back into a familiar. It isn’t affected by area effects and must be targeted separately to affect it, which requires knowledge that it’s a creature.</p>\n<p>This means you and your allies can heal or assist the familiar while most enemies stay unaware of its true nature. Creatures must attempt a @Check[type:perception|dc:20] check to @UUID[Compendium.pf2e.actionspf2e.Item.Seek] to realize a it is actually a familiar. Your familiar can still communicate its feelings empathically. Transforming the familiar between forms is a 1-minute activity that has the concentrate trait.</p>"
```

```suggestion
            "value": "<p>Your familiar can transform into a mark you carry on your flesh, typically seeming like a birthmark, tattoo, or gem that vaguely resembles its normal form. When transformed, the familiar can't act except to turn back into a familiar. It isn't affected by area effects and must be targeted separately to affect it, which requires knowledge that it's a creature.</p>\n<p>This means you and your allies can heal or assist the familiar while most enemies stay unaware of its true nature. Creatures must attempt a @Check[type:perception|dc:20] check to @UUID[Compendium.pf2e.actionspf2e.Item.Seek] to realize a it is actually a familiar. Your familiar can still communicate its feelings empathically. Transforming the familiar between forms is a 1-minute activity that has the concentrate trait.</p>"
```
Sneaky rsquo, be sure to use the data entry script to help keep these from cropping up.